tests/rt.sh

```diff
@@ -368,7 +375,9 @@ elif [[ $MACHINE_ID = s4 ]]; then
   ROCOTOCOMPLETE=$(which rocotocomplete)
   ROCOTO_SCHEDULER=slurm
 
-  ECFLOW_START=/opt/ecflow/5.6.0/bin/ecflow_start.sh
+  module use /data/prod/jedi/spack-stack/modulefiles
```

Comment:
same qustion here. is it OK ufs-weather-model gets support from jedi for ecflow update?

Reply:
@aerorahul do you know an answer to this?

Reply:
Yes, we have an agreement on this described on the spack-stack project charter (https://github.com/JCSDA/spack-stack/blob/develop/project_charter.md).